contracts/Migrator.sol

```diff
@@ -7,6 +7,8 @@ import { IERC20Like } from "./interfaces/Interfaces.sol";
 
 contract Migrator {
```

Comment:
Let's have this contract inherit the interface and update the IMigrator interface to include the new scalar parm